application/src/main/java/org/thingsboard/server/service/cf/DefaultCalculatedFieldCache.java

```diff
@@ -117,7 +117,11 @@ public CalculatedFieldCtx getCalculatedFieldCtx(CalculatedFieldId calculatedFiel
                     CalculatedField calculatedField = getCalculatedField(calculatedFieldId);
                     if (calculatedField != null) {
                         ctx = new CalculatedFieldCtx(calculatedField, tbelInvokeService, apiLimitService);
-                        ctx.init();
+                        try {
+                            ctx.init();
```

Comment:
just remove this init here. We don't use it for processing, only to check the arguments and links